### ISSUE TITLE
ServerSync Not working!

### ISSUE THREAD

**Issue by cj89898** [lorddusk/HQM#167]
I am trying to edit the quests for a modpack that I banned some items for. I do this but serversync won't work! Nobody can open their questbooks.

Mod version: 4.4.4
Modpack: Project Ozone v1.5.4S

**Comment by way2muchnoise** [comment]
1. Edit mode doesn't work in servers
2. Did you turn questing mode on?
3. What do you mean by server sync?

**Comment by cj89898** [comment]
1. I edited the files in singleplayer then transfered.
2. Questing mode is on.
3. There is an option in the config called serversync.

`# Configuration file

general {
    # Always display the tier name, instead of the individual bag's name, when opening a reward bag.
    B:AlwaysUseTierNameForRewardTitles=false

```
# If set to true, new worlds will automatically activate Hardcore mode
B:"Auto-start hardcore mode"=false

# If set to true, new worlds will automatically activate Questing mode
B:"Auto-start questing mode"=true

# How many lives players should start with.
I:"Default lives"=255

# Use this to set the maximum lives obtainable (Max 255)
I:MaxLives=255

# Allow every single player in a party to claim the reward for a quest. Setting this to false will give the party one set of rewards to share.
B:MultiReward=true

# Enable or disable sending a status message if Hardcore Questing mode is off
B:NoHardcoreMessage=true

# Display an interface with the contents of the reward bag when you open it.
B:RewardInterface=true

# Define in seconds how long the rot timer is.
I:RotTime=120

# Turn on/off the rot timer.
B:RotTimer=false

# If this is set to true, the server will send the quests to clients connecting to it.
B:ServerSync=true <-------

# Use this if you want the book to spawn on create world
B:SpawnBook=true
```

}

`

**Comment by way2muchnoise** [comment]
Any errors on either client or server?

**Comment by way2muchnoise** [comment]
Closed due to no longer supporting 1.7.10, please check if this problem persists in the latest 1.9.4 version.
